techmatters/flex-plugins
# Fix flex deploy slack message & update UI version

### Checklist

### Other Related Issues

None

### Verification steps

Run the Flex Deploy workflow. Ensure that the slack message is written and the ui version is correct in the service configuration (you can call the GET service config endpoint https://flex-api.twilio.com/v1/Configuration with correct auth creds

### AFTER YOU MERGE

1. Cut a release tag using the Github workflow. Wait for it to complete and notify in the #aselo-deploys Slack channel.
2. Comment on the ticket with the release tag version AND any additional instructions required to configure an environment to test the changes.
3. Only then move the ticket into the QA column in JIRA

You are responsible for ensuring the above steps are completed. If you move a ticket into QA without advising what version to test, the QA team will assume the latest tag has the changes. If it does not, the following confusion is on you! :-P
